### ISSUE TITLE
is there an uninstalling routine ?

### ISSUE THREAD

**Issue by aracloud** [mzijlstra/hlCursors#2]

**Comment by mzijlstra** [comment]
No, but people usually just "install" by moving the files it makes into a directory either in ~/.local/share/icons/   ("installed for the user"), or system wide in a directory in: /usr/share/icons/

Which makes uninstalling as easy as removing the directory
